packages/webcert/src/components/error/modals/ComplementaryCertificateExists.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
```

Comment:
Thare have been some changes in these error modals in the master branch (by Michaela) that you need to make sure they are not overwritten by the merge
https://github.com/sklintyg/frontend/pull/685/

Reply:
Good catch, I fixed that and manually went through every file to see that I haven't missed anything else. Should be good now :)